Jenkinsfile

```diff
@@ -4,8 +4,42 @@ properties(
   ]
 )
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
 node('buildvm-devops') {
-  sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
-    echo "Bruno's branch split pipeline"
+  try {
+    deleteDir()
+    set_workspace()
+    stage('clone') {
+      dir('aos-cd-jobs') {
+        checkout scm
+        sh 'git checkout master'
+      }
+    }
+    stage('run') {
+      sshagent(['openshift-bot']) { // git repo privileges stored in Jenkins credential store
+        sh '''\
+virtualenv env/
+env/bin/pip install gitpython
+export PYTHONPATH=$PWD/aos-cd-jobs
+env/bin/python aos-cd-jobs/aos_cd_jobs/pruner.py
```

Comment:
I think it's more appropriate to activate it. I don't know for certain that it's equivalent to use the `python` symlink in the `virtualenv` dir, as the activation also changes `$PATH` and `$PYTHON_HOME`. You may find the "normal" activate script to fail in Jenkins as the shell has no `$PS1`, so in other jobs we do:

```
export VIRTUAL_ENV="$( pwd )/env"
export PATH="${VIRTUAL_ENV}/bin:${PATH}"
unset PYTHON_HOME
```